cypress/integration/router_spec.js

```diff
@@ -0,0 +1,278 @@
+    cy.contains('Ny artikkel');
+
+    // Polls
+    openMenu();
+    cy.get(c('Search__navigationFlex')).within(() => {
+      cy.get(a('/polls')).click();
+    });
+    cy.url().should('contain', '/polls');
+    cy.contains('Avstemninger');
+
+    // Companies
+    openMenu();
+    cy.get(c('Search__navigationFlex')).within(() => {
+      cy.get(a('/companies')).click();
+    });
+    cy.url().should('contain', '/companies');
+    cy.contains('Bedrifter');
+
+    // Gallery
+    openMenu();
+    cy.get(c('Search__navigationFlex')).within(() => {
+      cy.get(a('/photos')).click();
+    });
+    cy.url().should('contain', '/photos');
+    cy.contains('Albumer');
+
+    // Interestgroups
+    openMenu();
+    cy.get(c('Search__navigationFlex')).within(() => {
+      cy.get(a('/interestgroups')).click();
+    });
+    cy.url().should('contain', '/interestgroups');
+    cy.contains('Interessegrupper');
+
+    // Joblistings
+    openMenu();
+    cy.get(c('Search__navigationFlex')).within(() => {
+      cy.get(a('/joblistings')).click();
+    });
+    cy.url().should('contain', '/joblistings');
+    cy.contains('Søknadsfrist');
+
+    // Contacs
+    openMenu();
+    cy.get(c('Search__navigationFlex')).within(() => {
+      cy.get(a('/contact')).click();
+    });
+    cy.url().should('contain', '/contact');
+    cy.contains('Kontaktskjema for Abakus');
+
+    // Meetings
+    openMenu();
+    cy.get(c('Search__navigationFlex')).within(() => {
+      cy.get(a('/meetings')).click();
+    });
+    cy.url().should('contain', '/meetings');
+    cy.contains('Dine Møter');
+
+    // About
+    openMenu();
+    cy.get(c('Search__navigationFlex')).within(() => {
+      cy.get(a('/pages/info-om-abakus')).click();
+    });
+    cy.url().should('contain', '/pages/info-om-abakus');
+    cy.contains('Generelt');
+
+    // Podcast
+    openMenu();
+    cy.get(c('Search__navigationFlex')).within(() => {
+      cy.get(a('/podcasts')).click();
+    });
+    cy.url().should('contain', '/podcasts');
+    cy.contains('Podcasts');
+
+    // Profile
+    openMenu();
+    cy.get(c('Search__navigationFlex')).within(() => {
+      cy.get(a('/users/me')).click();
+    });
+    cy.url().should('contain', '/users/me');
+    cy.contains('Brukerinfo');
+
+    // Quotes
+    openMenu();
+    cy.get(c('Search__navigationFlex')).within(() => {
+      cy.get(a('/quotes/?filter=all')).click();
+    });
+    cy.url().should('contain', '/quotes/?filter=all');
+    cy.contains('Just do it!');
+
+    // Tags
+    openMenu();
+    cy.get(c('Search__navigationFlex')).within(() => {
+      cy.get(a('/tags')).click();
+    });
+    cy.url().should('contain', '/tags');
+    cy.contains('lorem');
+  });
+
+  it('should be able to logg out', () => {
+    cy.visit('/');
+    cy.get(c('buttonGroup')).within(() => {
+      cy.get('img[alt="user"]').click();
+    });
+
+    // Logg out
+    cy.get(c('Dropdown')).within(() => {
+      cy.get(c('Button')).click();
```

Comment:
Is there a better way to select this button? It looks very generic and could easily fail if we add more options to the dropdown

Reply:
Yeah, that sucked 🙃  I'ma switch that out with `.contains('Logg ut')`